observability/build.gradle

```diff
@@ -20,3 +24,13 @@ dependencies {
 
     signature "org.codehaus.mojo.signature:java18:1.0@signature"
 }
+
+configureProtoCompilation()
+
+jacocoTestReport {
```

Comment:
Was this related to addition of LogRecord and proto files? Or is there another reason to add this?

Reply:
Yes, this was for LogRecord and proto files and also modeled on old grpc-java projects. I will remove.

Reply:
You can leave it in if that's the right thing to do i.e. if all projects should have jacocoTestReport. Pls check if something is mentioned in the build/compile docs